Cartfile

```diff
@@ -1 +1 @@
-github "CanalTP/NavitiaSDKUX_ios" == 2.1.0
+github "CanalTP/NavitiaSDKUX_ios" "develop"
```

Comment:
You need to specify the version instead of the branch name

Reply:
The version doesn't exist yet